doc/howto/network_zones.md

```diff
@@ -53,7 +53,11 @@ You can check the records that are generated with your zone setup with the `dig`
 This assumes that `core.dns_address` was set to `<DNS_server_IP>:<DNS_server_PORT>`.
 For example, running `dig @<DNS_server_IP> -p <DNS_server_PORT> axfr lxd.example.net` might give the following output:
 
-```bash
+```{terminal}
+:input: dig @192.0.2.200 -p 1053 axfr lxd.example.net
```

Comment:
Does this example IP make sense, or should it be one that appears in the output?

Reply:
I'm not sure I understand the question? What output?

Reply:
Yes that makes sense to me.